src/runtime_src/core/pcie/emulation/common_em/config.h

```diff
@@ -262,6 +262,7 @@ namespace xclemulation{
   std::string getXclbinVersion(const axlf* top);
   std::string getVivadoVersion();
   void checkXclibinVersionWithTool(const xclBin *header);
+  int getIPName2Index(const char *name,const char* ipLayoutbuf);
```

Comment:
Comments, comments,...

Reply:
yes, added comments and addressed your suggestions.